LeaderboardBackend/Models/Entities/User.cs

```diff
@@ -80,3 +91,26 @@ public override int GetHashCode()
         return HashCode.Combine(Id, Username, Email);
     }
 }
+
+public class UserEntityTypeConfig : IEntityTypeConfiguration<User>
+{
+    public void Configure(EntityTypeBuilder<User> builder)
+    {
+        builder.Property(x => x.Username)
+            .UseCollation(ApplicationContext.CASE_INSENSITIVE_COLLATION);
+
+        builder.Property(x => x.Email)
+            .UseCollation(ApplicationContext.CASE_INSENSITIVE_COLLATION);
+
+        builder.HasData(
+            new User
+            {
+                Id = User.s_seedAdminId,
+                Role = UserRole.Administrator,
+                Email = "omega@star.com",
+                Password = "$2a$11$tNvA94WqpJ.O7S7D6lVMn.E/UxcFYztl3BkcnBj/hgE8PY/8nCRQe", // "3ntr0pyChaos"
+                Username = "Galactus"
+            }
+        );
```

Comment:
It seeds the Postgres DB with an admin, so that we can run operations easier.

Reply:
That shouldn't be done here. This literally means that everyone has admin because this code is public. This needs to be handled securely by the database admin.

Reply:
Yeah, I'm aware. Figured that since this project's an MVP it's fine; it's quicker to develop.

Reply:
I don't think we should be committing seed data to the repo (unless it's for a local test env). This needs to be handled by whoever's in charge of the database. Everyone who needs admin access should make an account and follow the flow to be promoted to admin. Nobody should be coded to have admin.

Reply:
Since the seeding was not added in this PR (the `HasData` part was only moved from `ApplicationContext`) imma ignore it for now

Can you open an issue for this?

Reply:
Will do.